docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -659,12 +668,99 @@ def configure_optimizers(self):
         return [optim], [scheduler]
 
 
+class MAEModel(BenchmarkModule):
+    def __init__(self, dataloader_kNN, num_classes):
+        super().__init__(dataloader_kNN, num_classes)
+        
+        decoder_dim = 512
+        vit = torchvision.models.vit_b_32(pretrained=False)
+
+        self.warmup_epochs = 40 if max_epochs >= 800 else 20
+        self.mask_ratio = 0.75
+        self.patch_size = vit.patch_size
+        self.sequence_length = vit.seq_length
+        self.mask_token = nn.Parameter(torch.zeros(1, 1, decoder_dim))
+        self.backbone = masked_autoencoder.MAEBackbone.from_vit(vit)
+        self.decoder = masked_autoencoder.MAEDecoder(
+            seq_length=vit.seq_length,
+            num_layers=1,
+            num_heads=16,
+            embed_input_dim=vit.hidden_dim,
+            hidden_dim=decoder_dim,
+            mlp_dim=decoder_dim * 4,
+            out_dim=vit.patch_size ** 2 * 3,
+            dropout=0,
+            attention_dropout=0,
+        )
+        self.criterion = nn.MSELoss()
+
+    def random_mask(self, images):
```

Comment:
This is a bit unnecessary to put it into its own function don't you think? Takes away from the "simple-to-use"

Reply:
I agree, changed it :)